### ISSUE TITLE
FTBFS: roff.rb:165:in `block_filter': undefined method `position' for #<Nokogiri::XML::Element:0x0000559940a0a8a0> (NoMethodError)

### ISSUE THREAD

**Issue by jean-christophe-manciot** [git-lfs/git-lfs#3541]
Ubuntu 19.04 disco
git-lfs v2.7.0
ronn 0.8.0-1

Building git-lfs in a disco chroot with ```debuild -i -I --no-sign --build=binary -j1``` leads to:
```
...
ronn  -r --pipe < docs/man/git-lfs.1.ronn > man/git-lfs.1
/usr/lib/ruby/vendor_ruby/ronn/roff.rb:165:in `block_filter': undefined method `position' for #<Nokogiri::XML::Element:0x0000559940a0a8a0> (NoMethodError)
	from /usr/lib/ruby/vendor_ruby/ronn/roff.rb:88:in `block in block_filter'
	from /usr/lib/ruby/vendor_ruby/nokogiri/xml/node_set.rb:204:in `block in each'
	from /usr/lib/ruby/vendor_ruby/nokogiri/xml/node_set.rb:203:in `upto'
	from /usr/lib/ruby/vendor_ruby/nokogiri/xml/node_set.rb:203:in `each'
	from /usr/lib/ruby/vendor_ruby/ronn/roff.rb:88:in `block_filter'
	from /usr/lib/ruby/vendor_ruby/ronn/roff.rb:160:in `block_filter'
	from /usr/lib/ruby/vendor_ruby/ronn/roff.rb:88:in `block in block_filter'
	from /usr/lib/ruby/vendor_ruby/nokogiri/xml/node_set.rb:204:in `block in each'
	from /usr/lib/ruby/vendor_ruby/nokogiri/xml/node_set.rb:203:in `upto'
	from /usr/lib/ruby/vendor_ruby/nokogiri/xml/node_set.rb:203:in `each'
	from /usr/lib/ruby/vendor_ruby/ronn/roff.rb:88:in `block_filter'
	from /usr/lib/ruby/vendor_ruby/ronn/roff.rb:91:in `block_filter'
	from /usr/lib/ruby/vendor_ruby/ronn/roff.rb:18:in `initialize'
	from /usr/lib/ruby/vendor_ruby/ronn/document.rb:245:in `new'
	from /usr/lib/ruby/vendor_ruby/ronn/document.rb:245:in `to_roff'
	from /usr/lib/ruby/vendor_ruby/ronn/document.rb:240:in `convert'
	from /usr/bin/ronn:220:in `block (2 levels) in <main>'
	from /usr/bin/ronn:199:in `each'
	from /usr/bin/ronn:199:in `block in <main>'
	from /usr/bin/ronn:184:in `each'
	from /usr/bin/ronn:184:in `<main>'
make[2]: *** [Makefile:484: man/git-lfs.1] Error 1
```
Is this a missing build dependency?

[git-lfs_2.7.0+19.04_amd64.build.log](https://github.com/git-lfs/git-lfs/files/2893174/git-lfs_2.7.0%2B19.04_amd64.build.log)
.

**Comment by bk2204** [comment]
Hey, thanks for writing in.

This appears to be a bug in `ronn` (apjanke/ronn-ng#24). It's been reported to Debian in https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=921637, and when it's fixed there, it will likely make its way into Ubuntu as well.

Sorry I don't have better news for you.

**Comment by ssgelm** [comment]
@jean-christophe-manciot FYI I requested that the fixed version of this be pulled into disco: https://bugs.launchpad.net/ubuntu/+source/ruby-ronn/+bug/1817834.  Hopefully that will happen before the freeze.

**Comment by ssgelm** [comment]
@jean-christophe-manciot This should be fixed now in disco with ronn 0.8.0-2 (https://packages.ubuntu.com/disco/ronn)
